### PR TITLE
feat(FWS-75): Ability to View Trainer All Appointments

### PR DESCRIPTION
**Pull request by AlexeiTimbro**
**JIRA: link to jira ticket**
https://champlainsaintlambert.atlassian.net/browse/FWS-75

**What is the ticket about and why are we making this change?**

This ticket is about being able to view all the appointments in the system from the personal trainer panel by the trainer in the system. The function has also been added to the admin panel as well since the admin can view all the appointments as well. The purpose of this ticket was to allow the personal trainer to manage all the appointments that are either scheduled, completed or cancelled in the system.

**What are the various changes and what other modules do those changes affect?**
 
-4 types of testing
-Selenium Test
-backend for the get-all appointments
-Front end for the get-all appointments
- Trainer Panel view is completely working and done visually
- Added the Account box view in the Trainer panel
- Added the Appointment box view in the Admin Panel

**UI Change:**

After: (Trainer Panel View)
image

After clicking the Appointment box:
image

After clicking the Account box:
image

### CODE REVIEW

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                                <th>Availability Id</th>
+                                <th>Admin Id</th>
+                                <th>Service Id</th>
+                                <th>Status</th>
+                                <th>Location</th>
+                            </tr>
+                            </thead>
+                            <tbody>
+                            {appointments.map(appointments => (
+                                <tr key={appointments.id}>
```
Comment (Eli293) [inline fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js:107]: What does this do?

**Review by Eli293 — APPROVED** [review]
Looks good.

File: fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js (hunk 1 above)
Comment (AlexeiTimbro) [reply fitwsarah_frontend/src/views/AdminPanelPage/AdminAppointments.js:107]: This essentially maps every appointment in the body to their respective "appointmentId".

**Review by Abdur-Siddiqui — APPROVED** [review]
Looks good, good job

**Pull request merged**